module #3. REST API Advanced/src/main/java/com/epam/esm/dto/OrderDto.java

```diff
@@ -0,0 +1,29 @@
+package com.epam.esm.dto;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+import lombok.NoArgsConstructor;
+import org.springframework.hateoas.RepresentationModel;
+
+import java.math.BigDecimal;
+import java.time.LocalDateTime;
+
+@Data
+@AllArgsConstructor
+@NoArgsConstructor
+@Builder
+@EqualsAndHashCode(callSuper = false)
+public class OrderDto extends RepresentationModel<OrderDto> {
```

Comment:
Mostly at programming we choose "receipt" instead of order because order usually is reserved word